.chloggen/keep_going_if_collector_crd_is_not_present.yaml

```diff
@@ -0,0 +1,17 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: webhook
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Do not exit if the collector CRD is missing.
```

Comment:
I would prefer to add more information about this because right now, this is done by design, not by mistake.

Reply:
added more to the changelog